Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
-        precisemaneuver_keybindings_progdec             = Decrement prograde
-        precisemaneuver_keybindings_progzero            = Zero prograde
-        precisemaneuver_keybindings_norminc             = Increment normal
-        precisemaneuver_keybindings_normdec             = Decrement normal
-        precisemaneuver_keybindings_normzero            = Zero normal
-        precisemaneuver_keybindings_radinc              = Increment radial
-        precisemaneuver_keybindings_raddec              = Decrement radial
-        precisemaneuver_keybindings_radzero             = Zero radial
-        precisemaneuver_keybindings_timeinc             = Increment time
-        precisemaneuver_keybindings_timedex             = Decrement time
-        precisemaneuver_keybindings_circularize         = Circularize orbit
-        precisemaneuver_keybindings_turnup              = Turn orbit up
-        precisemaneuver_keybindings_turndown            = Turn orbit down
-        precisemaneuver_keybindings_pageinc             = Change increment (w/Alt)
-        precisemaneuver_keybindings_pageconics          = Change orbit mode (w/Alt)
-        precisemaneuver_keybindings_encfocus            = Focus on next encounter
-        precisemaneuver_keybindings_patchesmore         = Show more patches
-        precisemaneuver_keybindings_patchesless         = Show fewer patches
-        precisemaneuver_keybindings_timex10             = Toggle x10 time
-        precisemaneuver_keybindings_nextnode            = Next maneuver node
-        precisemaneuver_keybindings_prevnode            = Prev maneuver node
-        precisemaneuver_keybindings_delnode             = Delete maneuver node
-                
-        precisemaneuver_date_format                     = Year <<1>>, Day <<2>>, <<3>>:<<4>>:<<5>>
-        precisemaneuver_date_format_compact             = Y<<1>> D<<2>> <<3>>:<<4>>:<<5>>
-                
-        precisemaneuver_node                            = Node
+   es-es {
+        precisemaneuver_caption                         = MANIOBRA PRECISA
+        precisemaneuver_keybindings_caption             = ATAJOS DE TECLADO MANIOBRA PRECISA
+                
+        precisemaneuver_mainmenu_enable                 = Habilitar
+        precisemaneuver_mainmenu_keybindings            = Establecer atajos
+        precisemaneuver_mainmenu_background             = Fondo
+        precisemaneuver_mainmenu_scale                  = Scala de la GUI
+        precisemaneuver_mainmenu_components             = COMPONENTES
+                
+        precisemaneuver_module_pager                    = Buscador de Maniobras
+        precisemaneuver_module_saver                    = Plantillas de Maniobras
+        precisemaneuver_module_precise_input            = Entrada precisa
+        precisemaneuver_module_orbit_tools              = Herramientas de Órbita
+        precisemaneuver_module_gizmo                    = Gadget de Maniobra
```

Comment:
gadget is also an english word, what's the meaning of gizmo in this case? i don't think we have a word for gizmo but maybe we can replace it with an spanish alternative